### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by CMonnin**
@joshunrau code review finished and addressed.

### CODE REVIEW

File: .github/workflows/node-build.yml
Comment (joshunrau) [inline .github/workflows/node-build.yml:1]: you want to deploy from dev?

File: src/pictureNamingTask.ts [hunk 1]
```diff
@@ -72,165 +64,157 @@ function getRandomElementWithSeed(array: any[]) {
...
-    );
-    console.log("filtered");
-    console.log(imgList);
-    let result = getRandomElementWithSeed(imgList);
-    return result;
-  } catch (error) {
-    console.error("Error fetching and parsing data:", error);
-  }
+  let imgList: ExperimentImage[] = imageDB;
+  imgList = imgList.filter(
```
Comment (joshunrau) [inline src/pictureNamingTask.ts]: here, you can just set `imgList = imgDb.filter`

**Review by joshunrau — CHANGES_REQUESTED** [review]
awesome!

File: .github/workflows/node-build.yml
Comment (CMonnin) [reply .github/workflows/node-build.yml:1]: nah, this was from testing. will remove.

File: src/pictureNamingTask.ts (hunk 1 above)
Comment (CMonnin) [reply src/pictureNamingTask.ts]: true!

Comment (CMonnin) [reply src/pictureNamingTask.ts]: see 8d21bb4

File: .github/workflows/node-build.yml
Comment (CMonnin) [reply .github/workflows/node-build.yml:1]: see  8d21bb4

**Pull request merged**